es-ds-components/components/es-verification-code.vue

```diff
@@ -0,0 +1,89 @@
+<script setup lang="ts">
+import InputOtp from 'primevue/inputotp';
+
+const model = defineModel<string>();
+const verificationCode = ref();
+
+const props = defineProps({
+    charCount: {
+        type: Number,
+        default: 5,
+    },
+});
+
+const count = ref(props.charCount);
+watch(model, (newVal) => {
+    verificationCode.value = newVal;
+});
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const updateCode = (newValue: any) => {
+    model.value = newValue;
+};
+</script>
+
+<template>
+    <div>
+        <input-otp
+            v-model="verificationCode"
+            class="custom-otp"
```

Comment:
On the docs page, it looks like you're passing `isSubmitInProgress` in through a `disabled` prop to this component, but then not doing anything with it, so the inputs don't get disabled during the simulated server request. If you add a `v-bind="$attrs"` here on `<input-otp>`, that should pass anything else through, including the `disabled` prop.